types/plotly.js/lib/traces/sankey.d.ts

```diff
@@ -0,0 +1,102 @@
+import { PlotData, DataTitle, Datum, HoverLabel } from '../../index';
+
+export type SankeyColor = string | number;
+export type SankeyColors = Array<SankeyColor | null | undefined>;
+
+export interface SankeyFont {
+    family: string | string[];
+    size: number | number[];
+    color: SankeyColor | SankeyColors;
+}
+
+export interface SankeyDataTitle {
+    font: Partial<SankeyFont>;
+    title: string;
+}
+
+export type SankeyOrientation = 'v' | 'h';
+
+export interface SankeyHoverLabel {
+    bgcolor: SankeyColor | SankeyColors;
+    bordercolor: SankeyColor | SankeyColors;
+    font: SankeyFont;
+    align: HoverLabel['align'] | Array<HoverLabel['align']>;
+    namelength: number | number[];
+}
+
+export interface SankeyDomain {
+    row: number;
+    column: number;
+    x: number[];
+    y: number[];
+}
+
+export interface SankeyNode {
+    color: SankeyColor[];
+    customdata: Datum[];
+    groups: SankeyNode[];
+    hoverinfo: 'all' | 'none' | 'skip';
+    hoverlabel: Partial<SankeyHoverLabel>;
+    hovertemplate: string[];
+    label: Datum[];
+    line: Partial<{
+        color: SankeyColor;
+        width: number;
+    }>;
+    pad: number;
+    thickness: number;
+    x: number[];
+    y: number[];
+}
+
+export interface SankeyColorscale {
+    cmax: number;
+    cmin: number;
+    colorscale: Array<[number, string]>;
+    label: string;
+    name: string;
+    templateitemname: string;
+}
+
+export interface SankeyLink {
+    arrowlen: number;
+    color: SankeyColor | SankeyColor[];
+    colorscale: Partial<SankeyColorscale>;
+    customdata: Datum[];
+    hoverinfo: 'all' | 'none' | 'skip';
+    hoverlabel: Partial<SankeyHoverLabel>;
+    hovertemplate: string | string[];
+    label: Datum[];
+    line: Partial<{
+        color: SankeyColor;
+        width: number;
+    }>;
+    source: number[];
+    target: number[];
+    value: number[];
+}
+
+export interface SankeyData {
+    type: string;
```

Comment:
should this read:
```ts
```suggestion
    type: 'sankey';
```